src/session.c

```diff
@@ -325,6 +334,7 @@ static CURL *session_get_handle(bool new_handle) {
     log_print(LOG_INFO, SECTION_ENHANCED, "Opening cURL session -- fusedav.%s.sessions:1|c", filesystem_cluster);
     log_print(LOG_INFO, SECTION_SESSION_DEFAULT, "Opening cURL session");
     session = curl_easy_init();
+    update_session_count(true);
```

Comment:
What if session == NULL? I don't think we'll ever end up decrementing `session_count`.